src/Testcontainers.Ollama/OllamaBuilder.cs

```diff
@@ -0,0 +1,121 @@
+namespace Testcontainers.Ollama
+{
+    /// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+    [PublicAPI]
+    public sealed class OllamaBuilder : ContainerBuilder<OllamaBuilder, OllamaContainer, OllamaConfiguration>
+    {
+        /// <summary>
+        /// Gets the default port of the Ollama API.
+        /// </summary>
+        public const int DefaultPort = 11434;
+        
+        /// <summary>
+        /// Default image name and version tag.
+        /// </summary>
+        public const string OllamaImage = "ollama/ollama:0.1.22";
+    
+        /// <summary>
+        /// Default volume path.
+        /// </summary>
+        public const string DefaultVolumePath = "/root/.ollama";
+    
+        /// <summary>
+        /// Default volume name.
+        /// </summary>
+        public const string DefaultVolumeName = "ollama-volume";
+
+        /// <summary>
+        /// The default model name for the OllamaBuilder.
+        /// </summary>
+        public const string DefaultModelName = OllamaModels.Llama2;
```

Comment:
Models are just like images and should be provided as a string rather than having an specific type. It will also reduce the amount of PRs just to keep it up-to-date